projects/proposals/z:OSMF Workflows.md

```diff
@@ -0,0 +1,60 @@
+* Name of project (must be unique within OMP)
+z/OSMF Workflows
+* Project description (what it does, why it is valuable, origin and history)
+
+This projects enables customers to share z/OSMF workflows. z/OSMF workflows modernizes the system management tasks performed day to day by z/OS System Programmers. Sharing of z/OSMF workflows for system management will help customers to learn from each other.
+
+* Statement on alignment with OMP charter mission
+
+OMP is for mainframe cooperation and it's helpful to fulfill our mission to collaborate on z/OSMF workflows development.
+
+* Sponsor from TSC (sponsor helps mentor projects)
+
+John Mertic
+
+* Preferred maturity level (see [project stages](../../process/project_stages.md))
+
+Not still in incubate state but workflow technology is very mature.
+
+* License and contribution guidelines (refer to the [OMP guidelines](contribution_guidelines.md))
+
+No License.
```

Comment:
Assuming Apache 2 license for code and CC-BY-40 for documentation?

Reply:
z/OSMF workflows are in XML format. If Apache 2 is generic license that we should be using. I am ok with that. But technically, no license requirement with z/OSMF workflows.

Reply:
Gotcha - Apache 2 should cover this then.

Reply:
Apache 2.0, MIT or BSD would make sense from a license perspective.  It looks like this would be a source only release which would be XML documents.  

As a project, are there active developers building out workflows for release that are properly abstracted for use in a broad context or is this more of a "sharing platform".  The reason I ask is if the workflows are to be released normally they are tested and carry some level of development and testing prior to release.  

Is this what you are thinking about @shreeji818 ?

Reply:
Yes. z/OSMF workflows are xml documents. I added Apache 2.0.

This will be more of a "sharing platform". I don't expect workflows released would be abstracted at a level that it would work on any customer sites. I also expect that customer would release the workflows that they have tested on their platform but I don't think the workflow should work without any change if some other customer takes the workflow from community. May be there are some workflows that are generic and would work without any change. I don't have an example of such workflow yet.